### ISSUE TITLE
Job init failing with Parameter name: can't be prefixed with "ssm" when run on a new project

### ISSUE THREAD

**Issue by pcolazurdo** [aws/copilot-cli#1778]
I've found a few issues trying to create a new Job on an existing App.

When running ```copilot job init```, I get the following error:

```
✔ Wrote the manifest for job covid19-job at copilot/covid19-job/manifest.yml
Your manifest contains configurations like your container size and job schedule (@daily).

✘ get application : get application : ValidationException: Parameter name: can't be prefixed with "ssm" (case-insensitive). If formed as a path, it can consist of sub-paths divided by slash symbol; each sub-path can be formed as a mix of letters, numbers and the following 3 symbols .-_
```

The only way to move past this point is by running **copilot** with the specific app Name in the CLI like this

```
copilot job init --app modernization-lab
```

**Comment by iamhopaul123** [comment]
Hello @pcolazurdo. Sorry to hear you were getting this confusing error. This sounds like a bug to me. Normally you need to run `copilot app init` before creating any service/job, even though using an existing app. So I tried to reproduce the error, it seems like when I was running `copilot svc init` in a workspace without any app, it errors out with ```✘ could not find an application attached to this workspace, please run `app init` first```.

However, when I tried to run `copilot job init`, it didn't show the same error. I'll send a quick fix on this to make sure it errors with proper prompt in an early stage.

**Comment by efekarakus** [comment]
This fix was released just now in v1.1.0: https://github.com/aws/copilot-cli/releases/tag/v1.1.0!
